<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>tridimAppClass</class>
 <widget class="QMainWindow" name="tridimAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>622</width>
    <height>399</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>tridimApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="MyViewer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>361</width>
      <height>311</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="coord_x">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="coord_y">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="coord_z">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>10</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Add</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>50</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Remove</string>
    </property>
   </widget>
   <widget class="QPushButton" name="clear">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>50</y>
      <width>71</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Clear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="grid_cube">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>100</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Cube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rand_cube">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>140</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Cube</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rand_sphere">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>180</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Sphere</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rand_shell_sphere">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>220</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Shell Sphere</string>
    </property>
   </widget>
   <widget class="QPushButton" name="rand_hexagon">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>260</y>
      <width>141</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Random Hexagon</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>622</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MyViewer</class>
   <extends>QWidget</extends>
   <header>myviewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="tridimApp.qrc"/>
 </resources>
 <connections/>
</ui>
